### PR TITLE
Gyroscope and Accelerometer Fix

### PR DESCRIPTION
**Pull request by xXBenjaminYoungXx**
### Objectives:
- [x] Add sensor field names
- [x] Remove unessassary LUT elements
- [x] Adjust scale of accelerometer from g to m/s^2
- [x] Add comments relevant to decisions made
- [x] Update documentation (Overleaf) where nessassary

### CODE REVIEW

File: protos/NUgus.proto [hunk 1]
```diff
@@ -1347,286 +1346,35 @@ PROTO NUgus [
         # Accelerometer and Gyro are positioned 0.2m above torso base
         translation 0.000000 0.000000 0.2
         children [
-          # lis331dlh Accelerometer (See Section 1. of docs/Robot_Model_RoboCup_2021 for details on parameters)
+          #(See Section 1. of docs/Robot_Model_RoboCup_2021 for details on noise)
           # Axis: x forward, y left, z up
+          # Units are [m/s^2]
+          # The acelerometer measures the reaction forces over 3 axes in [m/s^2], thus,
+          # at rest the accelerometer should read +9.81 [m/s^2] in the z axis.
```
Comment (Bidski) [inline protos/NUgus.proto:1353]: Gravity should be down and z should be up!
```suggestion
          # at rest the accelerometer should read -9.81 [m/s^2] in the z axis.
```

**Review by Bidski — CHANGES_REQUESTED** [review]

File: protos/NUgus.proto (hunk 1 above)
Comment (TheCoolKidsTable) [reply protos/NUgus.proto:1353]: This is not how the accelerometer behaves in WeBots. It is based on the reaction force experienced by the accelerometer, thus, 9.81m/s^2 when z is up at rest.

Comment (Bidski) [reply protos/NUgus.proto:1353]: image

Comment (Bidski) [reply protos/NUgus.proto:1353]: https://cyberbotics.com/doc/reference/accelerometer

> An Accelerometer at rest with earth's gravity will indicate 1 g (9.81 m/s²) along the vertical axis. Note that the gravity can be specified in the gravity field in the WorldInfo node

I assume this means that if we set `gravity` to `-9.81` in WorldInfo we will get the behaviour we desire? Or will this just screw everything up?

**Review by Bidski — APPROVED** [review]
LGTM

**Review by TheCoolKidsTable — APPROVED** [review]
LGTM

**Pull request merged**